### PR TITLE
Update module k8s.io/api to v0.27.0 - abandoned

### PR DESCRIPTION
**Pull request by devex-sa**
This PR contains the following updates:

| Package | Type | Update | Change |
|---|---|---|---|
| [k8s.io/api](https://togithub.com/kubernetes/api) | require | minor | `v0.26.3` -> `v0.27.0` |

---

### Release Notes

<details>
<summary>kubernetes/api</summary>

### [`v0.27.0`](https://togithub.com/kubernetes/api/compare/v0.26.4...v0.27.0)

[Compare Source](https://togithub.com/kubernetes/api/compare/v0.26.4...v0.27.0)

### [`v0.26.4`](https://togithub.com/kubernetes/api/compare/v0.26.3...v0.26.4)

[Compare Source](https://togithub.com/kubernetes/api/compare/v0.26.3...v0.26.4)

</details>

---

### Configuration

📅 **Schedule**: Branch creation - At any time (no schedule defined), Automerge - At any time (no schedule defined).

🚦 **Automerge**: Enabled.

♻ **Rebasing**: Whenever PR becomes conflicted, or you tick the rebase/retry checkbox.

🔕 **Ignore**: Close this PR and you won't be reminded about this update again.

---

 - [ ] If you want to rebase/retry this PR, check this box

---

This PR has been generated by [Renovate Bot](https://togithub.com/renovatebot/renovate).

### CODE REVIEW

**Comment by devex-sa** [comment]
### ⚠ Artifact update problem

Renovate failed to update an artifact related to this branch. You probably do not want to merge this PR as-is.

♻ Renovate will retry this branch, including artifacts, only when one of the following happens:

 - any of the package files in this branch needs updating, or 
 - the branch becomes conflicted, or
 - you click the rebase/retry checkbox if found above, or
 - you rename this PR's title to start with "rebase!" to trigger it manually

The artifact failure details are included below:

##### File name: orchestrator/go.sum

```
Command failed: go get -d -t ./...
go: downloading github.com/aws/aws-sdk-go-v2/config v1.18.21
go: downloading github.com/aws/aws-sdk-go-v2/service/organizations v1.19.4
go: downloading github.com/aws/aws-sdk-go-v2 v1.17.8
go: downloading github.com/aws/aws-sdk-go v1.44.243
go: downloading k8s.io/api v0.27.0
go: downloading k8s.io/apimachinery v0.27.0
go: downloading k8s.io/client-go v0.26.3
go: downloading github.com/aws/aws-sdk-go-v2/credentials v1.13.20
go: downloading github.com/aws/aws-sdk-go-v2/internal/ini v1.3.33
go: downloading github.com/aws/aws-sdk-go-v2/feature/ec2/imds v1.13.2
go: downloading github.com/aws/aws-sdk-go-v2/service/sso v1.12.8
go: downloading github.com/aws/aws-sdk-go-v2/service/ssooidc v1.14.8
go: downloading github.com/aws/aws-sdk-go-v2/service/sts v1.18.9
go: downloading github.com/aws/smithy-go v1.13.5
go: downloading github.com/aws/aws-sdk-go-v2/internal/configsources v1.1.32
go: downloading github.com/gogo/protobuf v1.3.2
go: downloading github.com/google/gofuzz v1.1.0
go: downloading golang.org/x/net v0.8.0
go: downloading k8s.io/klog/v2 v2.90.1
go: downloading k8s.io/utils v0.0.0-20230209194617-a36077c30491
go: downloading github.com/aws/aws-sdk-go-v2/service/internal/presigned-url v1.9.26
go: downloading github.com/aws/aws-sdk-go-v2/internal/endpoints/v2 v2.4.26
go: downloading github.com/jmespath/go-jmespath v0.4.0
go: downloading sigs.k8s.io/structured-merge-diff/v4 v4.2.3
go: downloading gopkg.in/inf.v0 v0.9.1
go: downloading github.com/golang/protobuf v1.5.3
go: downloading github.com/google/gnostic v0.5.7-v3refs
go: downloading golang.org/x/time v0.0.0-20220210224613-90d013bbcef8
go: downloading github.com/davecgh/go-spew v1.1.1
go: downloading golang.org/x/term v0.6.0
go: downloading golang.org/x/oauth2 v0.0.0-20220223155221-ee480838109b
go: downloading github.com/go-logr/logr v1.2.3
go: downloading sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd
go: downloading github.com/json-iterator/go v1.1.12
go: downloading gopkg.in/yaml.v2 v2.4.0
go: downloading google.golang.org/protobuf v1.28.1
go: downloading k8s.io/kube-openapi v0.0.0-20230308215209-15aac26d736a
go: downloading gopkg.in/yaml.v3 v3.0.1
go: downloading golang.org/x/sys v0.6.0
go: downloading golang.org/x/text v0.8.0
go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: downloading github.com/modern-go/reflect2 v1.0.2
go: downloading sigs.k8s.io/yaml v1.3.0
go: downloading github.com/google/uuid v1.3.0
go: downloading github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
go: downloading google.golang.org/appengine v1.6.7
go: downloading github.com/go-openapi/jsonreference v0.20.1
go: downloading github.com/go-openapi/swag v0.22.3
go: downloading github.com/google/go-cmp v0.5.9
go: downloading github.com/emicklei/go-restful/v3 v3.9.0
go: downloading github.com/mailru/easyjson v0.7.7
go: downloading github.com/go-openapi/jsonpointer v0.19.6
go: downloading github.com/josharian/intern v1.0.0
github.com/dfds/aws-inventory-orchestrator/orchestrator/k8s imports
	k8s.io/client-go/kubernetes imports
	k8s.io/client-go/kubernetes/typed/resource/v1alpha1 imports
	k8s.io/api/resource/v1alpha1: cannot find module providing package k8s.io/api/resource/v1alpha1

```

**Review by avnes — APPROVED** [review]

**Review by avnes — CHANGES_REQUESTED** [review]
wait for Renovate to be happy

File: orchestrator/go.mod [hunk 1]
```diff
@@ -6,7 +6,7 @@ require (
 	github.com/aws/aws-sdk-go v1.44.243
 	github.com/aws/aws-sdk-go-v2/config v1.18.21
 	github.com/aws/aws-sdk-go-v2/service/organizations v1.19.4
-	k8s.io/api v0.26.3
+	k8s.io/api v0.27.0
 	k8s.io/apimachinery v0.26.3
 	k8s.io/client-go v0.26.3
```
Comment (emil2k) [inline orchestrator/go.mod]:
```suggestion
	k8s.io/client-go v0.27.0
```

File: orchestrator/go.mod [hunk 2]
```diff
@@ -6,7 +6,7 @@ require (
 	github.com/aws/aws-sdk-go v1.44.243
 	github.com/aws/aws-sdk-go-v2/config v1.18.21
 	github.com/aws/aws-sdk-go-v2/service/organizations v1.19.4
-	k8s.io/api v0.26.3
+	k8s.io/api v0.27.0
 	k8s.io/apimachinery v0.26.3
```
Comment (emil2k) [inline orchestrator/go.mod]:
```suggestion
	k8s.io/apimachinery v0.27.0
```

**Review by emil2k — CHANGES_REQUESTED** [review]

**Review by emil2k — APPROVED** [review]

**Comment by emil2k** [comment]
@avnes I've update the packages here which builds, but I don't know why snyk checks are failing and there is not much information on their UI, maybe you know?
Screenshot 2023-04-14 at 13 56 04

**Comment by devex-sa** [comment]
### Edited/Blocked Notification

Renovate will not automatically rebase this PR, because it does not recognize the last commit author and assumes somebody else may have edited the PR.

You can manually request rebase by checking the rebase/retry box above.

 ⚠ **Warning**: custom changes will be lost.

**Comment by devex-sa** [comment]
### Autoclosing Skipped

This PR has been flagged for autoclosing. However, it is being skipped due to the branch being already modified. Please close/delete it manually or report a bug if you think this is in error.

**Pull request closed without merging**